packages/perseus/src/widgets/radio/radio.ts

```diff
@@ -106,19 +105,16 @@ const transform = (
 ): RenderProps => {
     const choices = _choiceTransform(widgetOptions, strings, problemNum);
 
-    const numCorrect: number = _.reduce(
-        widgetOptions.choices,
-        function (memo, choice) {
-            return choice.correct ? memo + 1 : memo;
-        },
-        0,
-    );
```

Comment:
The main idea is to pull this logic out of `transform` and call it during the upgrade process. That way the widget can do all the pre-scoring things it needs to without answers (ie. render and be answerable).